enterprise/server/tasksize_model/tasksize_model.go

```diff
@@ -1,32 +1,309 @@
 package tasksize_model
 
 import (
+	"context"
+	"encoding/json"
+	"flag"
+	"io"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strconv"
+	"time"
+
 	"github.com/buildbuddy-io/buildbuddy/enterprise/server/remote_execution/platform"
+	"github.com/buildbuddy-io/buildbuddy/server/environment"
+	"github.com/buildbuddy-io/buildbuddy/server/util/grpc_client"
+	"github.com/buildbuddy-io/buildbuddy/server/util/log"
+	"github.com/buildbuddy-io/buildbuddy/server/util/status"
+	"golang.org/x/sync/errgroup"
 
 	repb "github.com/buildbuddy-io/buildbuddy/proto/remote_execution"
 	scpb "github.com/buildbuddy-io/buildbuddy/proto/scheduler"
+	tf_framework "github.com/buildbuddy-io/tensorflow-proto/tensorflow/core/framework"
+	tf "github.com/buildbuddy-io/tensorflow-proto/tensorflow_serving/apis"
+)
+
+var (
+	featuresConfigPath = flag.String("remote_execution.task_size_model.features_config_path", "", "Path pointing to features.json config file.")
+	servingAddress     = flag.String("remote_execution.task_size_model.serving_address", "", "Address pointing to TF server with saved Tensorflow models (cpu, mem).")
```

Comment:
I might be missing some context. What was the motivation for serving the models from a separate server?

Reply:
Tyler and I decided that the build tooling changes were getting to be too complicated. On top of needing to generate the go-tensorflow repo (which might be high maintenance if we want to upgrade TF versions), we would've also had to create a separate build of the app for Linux which doesn't depend on TF, since we don't currently support dynamically linking `.so` libraries in the standalone binary (since everything needs to be in the embedded FS, but we can't link from that embedded FS at runtime).